### ISSUE TITLE
Appsody operator 0.4.0 does not start if Knative operator is installed but not configured

### ISSUE THREAD

**Issue by kaczyns** [appsody/appsody-operator#251]
### What did you do?

Install Kabanero 0.8.0-rc.1 on a new OCP cluster.

### What did you expect to see?

The Appsody operator pod starts successfully

### What did you see instead?

This error:
```
{"level":"error","ts":1586375586.661519,"logger":"controller-runtime.source","msg":"if kind is a CRD, it should be installed before calling Start","kind":"Service.serving.knative.dev","error":"no matches for kind \"Service\" in version \"serving.knative.dev/v1alpha1\"","stacktrace":"github.com/go-logr/zapr.(*zapLogger).Error\n\tappsody-operator/vendor/github.com/go-logr/zapr/zapr.go:128\nsigs.k8s.io/controller-runtime/pkg/source.(*Kind).Start\n\tappsody-operator/vendor/sigs.k8s.io/controller-runtime/pkg/source/source.go:88\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func1\n\tappsody-operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:165\nsigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start\n\tappsody-operator/vendor/sigs.k8s.io/controller-runtime/pkg/internal/controller/controller.go:198\nsigs.k8s.io/controller-runtime/pkg/manager.(*controllerManager).startLeaderElectionRunnables.func1\n\tappsody-operator/vendor/sigs.k8s.io/controller-runtime/pkg/manager/internal.go:477"}
```

### Environment

* Appsody CLI version: N/A

* Appsody Stack: N/A

* Kubernetes version information: 1.16

### Possible solution

Seems that https://github.com/application-stacks/runtime-component-operator/issues/66 is the same issue.

Kabanero doesn't configure operators until all of the operators are installed.  So the Serverless (Knative) operator will be installed but not configured, before the Appsody operator is installed.

**Comment by leochr** [comment]
@kaczyns right, it's the same issue as https://github.com/application-stacks/runtime-component-operator/issues/66 you pointed out.

@arturdzm is in the process of fixing Appsody and Open Liberty operators.

**Comment by arthurdm** [comment]
@leochr / @arturdzm - I believe this has now been fixed in the 0.4.1 release?

**Comment by leochr** [comment]
Yes, fix included with 0.4.1. Open Liberty will include the fix in 0.5.0.
